lib/daru/io/exporters/csv.rb

```diff
@@ -5,16 +5,28 @@ module IO
     module Exporters
       module CSV
```

Comment:
@v0dro @zverok @lokeshh - Please have a look at the changes I've incorporated in the [exporters/excel.rb](https://github.com/athityakumar/daru-io/pull/16/files#diff-6440431570465cd21bbd3c56f6e2ea7c) file, in regards to the lazy-call feature. 

This now allows to have a call like `Daru::IO::Exporters::Excel.new(df, tempfile.path).write`